Database/index.sql

```diff
@@ -17,6 +17,9 @@ SET
 SET
     @DEVICE_EMPATICA := 2;
 
+SET

```

Comment:
If we have a left then we should have a right, but we have just a normal WearOS constant, check this and fix it or add a comment explaining the reason behind it.

Reply:
We call it like this because sometimes we have only one wear os, so its neutral, however, it's better if we call them right and left.